xcdat/spatial_avg.py

```diff
@@ -203,6 +203,27 @@ def _validate_axis(
 
         return axis
 
+    def _validate_domain_bounds(self, domain_bounds: xr.DataArray):
+        """Validates the ``bounds`` arg based on a set of criteria.
+
+        Parameters
+        ----------
+        domain_bounds: xr.DataArray
+            The bounds of an axis
+
+        Raises
+        ------
+        TypeError
+            If the ``bounds`` of a grid cell are not ordered low-to-high.
```

Comment:
I think ``bounds`` should be ``domain_bounds``?

Reply:
Correct